TouchID.ios.js

```diff
@@ -13,10 +13,14 @@ var ERRORS = require('./data/errors');
  */
 
 var TouchID = {
-  authenticate(callback) {
+  authenticate(reason, callback) {
+      // if user did not pass a reason or the reason was undefined
+      if(!reason){
```

Comment:
Can you add a space between the `if` keyword and the conditional.

Again, sorry for not having a eslint config.